source/ChromeDevTools/ChromeSession.cs

```diff
@@ -24,12 +25,13 @@ public class ChromeSession : IChromeSession
         private WebSocket _webSocket;
         private static object _Lock = new object();
 
-        public ChromeSession(string endpoint, ICommandFactory commandFactory, ICommandResponseFactory responseFactory, IEventFactory eventFactory)
+        public ChromeSession(string endpoint, ICommandFactory commandFactory, ICommandResponseFactory responseFactory, IEventFactory eventFactory, Action<Exception> onError)
```

Comment:
This should be optional.  If it's not provided, it should `throw` by default.

Reply:
Yes, it is a breaking change, but if we keep the throwing behaviour, there is no place to catch it - since it can occur on another thread while recieving event from chrome.
Are you sure you really want to keep this behaviour by default instead of forcing user to handle it?